package.json

```diff
@@ -1,5 +1,11 @@
 {
   "name": "vc-theme-b2b-vue",
+  "author": {
+    "email": "info@virtocommerce.com",
+    "name": "Virto Commerce",
+    "url": "https://virtocommerce.com"
+  },
+  "license": "VCOSL",
```

Comment:
https://docs.npmjs.com/cli/v10/configuring-npm/package-json#license
VirtoCommerce Open Source License don't have SPDX ID, so we need to add [LICENSE](https://github.com/VirtoCommerce/vc-platform/blob/dev/LICENSE) file and change this parameter to "SEE LICENSE IN LICENSE"